test/typescript/runtime.test.ts

```diff
@@ -0,0 +1,266 @@
+import cds, { User, Query } from '@sap/cds';
+import { struct } from '@sap/cds/apis/csn';
+
+import {
+  Service,
+  EventContext,
+  Request,
+  Event,
+  // User, >> already imported above
+
+  ApplicationService,
+  MessagingService,
+  DatabaseService,
+  RemoteService,
+
+  Association,
+  Composition,
+  // entity,
+  event,
+  // type,
+  array,
+  // struct,
+  service,
+
+} from '@sap/cds';
+
+
+describe('runtime tests', () => {
```

Comment:
I had to move the test code into a real test suite.  Otherwise test runners will complain.